pkg/migration/stop_old_master.go

```diff
@@ -0,0 +1,97 @@
+package migration
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func (m *azureMigrator) stopOldMasterComponents(ctx context.Context) error {
+	// Get old master node name.
+	nodeNames, err := m.getLegacyMasterNodeNames(ctx)
+	if err != nil {
+		return microerror.Mask(err)
+	}
+
+	m.logger.Debugf(ctx, "Found %d legacy nodes", len(nodeNames))
```

Comment:
```suggestion
	m.logger.Debugf(ctx, "found %d legacy nodes", len(nodeNames))
```

I wouldn't be against printing names and it can help with debugging if things go south with the labels.